lib/brainstem/dsl/block_field.rb

```diff
@@ -0,0 +1,69 @@
+require 'brainstem/dsl/configuration'
+require 'brainstem/dsl/field'
+
+module Brainstem
+  module DSL
+    class BlockField < Field
+      attr_reader :configuration
+
+      delegate :to_h, :keys, :has_key?, :each, to: :configuration
+
+      def initialize(name, type, options, parent_field = nil)
+        super(name, type, options)
```

Comment:
Woah, this subclass takes a different number of args than the superclass?! Weird man